src/main/java/uk/gov/companieshouse/model/Constants.java

```diff
@@ -22,9 +22,16 @@ private Constants() {
    /* Email */
    public static final String EMAIL_APP_ID = "dissolution-api";
    public static final String EMAIL_TOPIC = "email-send";
-   public static final String SUCCESSFUL_PAYMENT_EMAIL_SUBJECT = "Your application has been submitted";
+
+   public static final String SUCCESSFUL_PAYMENT_EMAIL_SUBJECT = "Your application to strike off and dissolve a company has been submitted";
    public static final String SUCCESSFUL_PAYMENT_MESSAGE_TYPE = "dissolution-payment-confirmation";
 
+   public static final String APPLICATION_ACCEPTED_EMAIL_SUBJECT = "Your application to strike off and dissolve a company has been accepted";
+   public static final String APPLICATION_ACCEPTED_MESSAGE_TYPE = "dissolution-submission-accepted";
```

Comment:
make these snake case - the payment confirmation email seems to be one of the only templates that uses hyphens.